yunwoo/LV_1/핸드폰-번호-가리기.js

```diff
@@ -0,0 +1,8 @@
+function solution(phone_number) {
+  const starQuantity = phone_number.length - 4;
+
+  return (
+    new Array(starQuantity).fill("*").join("") +
+    phone_number.substring(starQuantity)
+  );
+}
```

Comment:
생성자 함수를 이용하는 풀이법 잘 봤습니다!!👍
```suggestion
function solution(phone_number) {
  const starQuantity = phone_number.length - 4;

  return (
    new Array(starQuantity).fill("*").join("") +
    phone_number.substring(starQuantity)
  );
}
```

Reply:
- 👍 좋은방법은 아닌거 같긴 한데 이런방법 저런방법도 있다는걸 보여드리고 싶었습니다ㅎㅎ